.gitlab-ci.yml

```diff
@@ -598,12 +594,43 @@ release:deployment:tag:
       ./scripts/deploy-image.sh "${image[0]}" \'mainnet\' "$CI_REGISTRY_IMAGE";
       echo \'Deploying ECS service to mainnet\';
       '
+    - echo 'Waiting for seed nodes to update'
+    - >
+      nix-shell --arg ci true --run $'
+      ./scripts/wait-for-deploy.js mainnet;
+      '
   after_script:
     - rm -f "$REGISTRY_AUTH_FILE"
   rules:
     # Runs on tag pipeline where the tag is a release version
     - if: $CI_COMMIT_TAG =~ /^v[0-9]+\.[0-9]+\.[0-9]+$/
 
+release:integration:docker:
```

Comment:
The naming style was meant to be `stage:job:variant`.

So in the `integration:deployment`, we later have `integration:docker`.

So instead here after `release:deployment`, we should have `release:docker`.

At the very end there's `release:distribution`. The names here probably all needs to be refactored a bit.

But:

* `integration:deployment` - does the deployment to testnet
* `integration:...` - does tests relating to the testnet
* `integration:prerelease` - does the prerelease distribution
* `integration:merge` - does the merge
* `release:deployment` - deploys to mainnet
* `release:...` - does tests in relation to mainnet
* `release:distribution` - this does the final distribution

To match the naming I suggest `release:docker` to match `integration:docker`.

Reply:
Made the change.